homeassistant/components/growatt_server/sensor.py

```diff
@@ -867,4 +1028,5 @@ def update(self):
 
     def get_data(self, variable):
         """Get the data."""
+        _LOGGER.debug("The value for %s is: %s", variable, self.data.get(variable))
```

Comment:
How attached are you to this debug statement? Feels like it's going to produce a lot of output if enabled. Will it provide much added value for live systems if it is enabled?